src/application/deployer.go

```diff
@@ -9,52 +9,28 @@ import (
 	"github.com/Originate/exosphere/src/types"
 )
 
-// Deployer contains information needed to deploy the application
-type Deployer struct {
-	AppConfig      types.AppConfig
-	ServiceConfigs map[string]types.ServiceConfig
-	Logger         chan string
-	AppDir         string
-	HomeDir        string
-}
-
-// Start starts the deployment process
-func (d *Deployer) Start() error {
-	terraformDir := d.getTerraformDir()
-	terraformConfig := types.TerraformConfig{
-		AppConfig:      d.AppConfig,
-		ServiceConfigs: d.ServiceConfigs,
-		AppDir:         d.AppDir,
-		HomeDir:        d.HomeDir,
-		TerraformDir:   terraformDir,
-		RemoteBucket:   fmt.Sprintf("%s-terraform", d.AppConfig.Name),
-		LockTable:      "TerraformLocks",
-		Region:         "us-west-2", //TODO prompt user for this
-	}
-
-	err := aws.InitAccount(terraformConfig.RemoteBucket, terraformConfig.LockTable, terraformConfig.Region)
+// StartDeploy starts the deployment process
+func StartDeploy(deployConfig types.DeployConfig) error {
+	terraformDir := filepath.Join(deployConfig.AppDir, "terraform")
+	fmt.Printf("Setting up AWS account...\n\n")
```

Comment:
I'd prefer we don't have print statements in helper functions. Thoughts on passing in a logger?